benchmarks/free_surface_tractions/viscous/free_surface_viscous_cylinder_3D_loading.prm

```diff
@@ -0,0 +1,162 @@
+# This model applies a surface load (using the traction
+# boundary condition) on a free surface overlying 
+# a viscous box. It approximates the analytical solution
+# of Haskell (1935a) for an instantaneously applied
+# axisymmetric cylindrical load over a viscous half-space.
+
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 1500
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.5
+set Maximum time step                      = 10
+set Output directory                       = output_free_surface_viscous_cylinder_3D_loading
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+# Model geometry (500x500x500 km, 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set Z repetitions = 10
+    set X extent      = 500e3
+    set Y extent      = 500e3
+    set Z extent      = 500e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 1
+  set Initial global refinement          = 1
+  set Time steps between mesh refinement = 1
+  set Strategy				 = strain rate
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
+end
+
+subsection Free surface
+  set Free surface boundary indicators = top
+  set Surface velocity projection      = normal
+  set Additional tangential mesh velocity boundary indicators = left,right,front,back
+end
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, left, right, front, back
+end
+
+# Prescribe a fixed vertical traction on the top boundary
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top z: function
+end
+
+subsection Boundary traction model
+  subsection Function
+    set Variable names = x,y,z,t
+    set Function constants  = r0=100.e3, H0=1.e3, t1=1.e3, rhoi=900, g=9.8, t0=1.e3
+        # r0 is load radius, H0 is load height, t1 is time load is removed, 
+	# rhoi is density of ice/load, g is gravitational acceleration 
+    set Function expression = 0; 0; if ((x^2+y^2)<r0^2 && t<t1,-g*H0*rhoi, 0)
+  end
+end
+
+# Number and name of compositional fields
```

Comment:
Unless you use the `Viscoelastic` material model below, there is no need to specify these compositional fields.